### ISSUE TITLE
[Docs] Form inside modal

### ISSUE THREAD

**Issue by lonix1** [twbs/bootstrap#40491]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

One can have a form inside a modal. Docs are [here](https://getbootstrap.com/docs/5.3/components/modal/#varying-modal-content).

But that example is incorrect because the submit button is outside the form.

I think this:
```html
<div class="modal" ...>
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header"><!-- ... --></div>
      <div class="modal-body"><form><!-- ... --></form></div>            <-----
      <div class="modal-footer"><!-- ... --></div>
    </div>
  </div>
</div>
```

should be like this:
```html
<div class="modal" ...>
  <form class="modal-dialog">                                             <-----
    <div class="modal-content">
      <div class="modal-header"><!-- ... --></div>
      <div class="modal-body"><!-- ... --></div>
      <div class="modal-footer"><!-- ... --></div>
    </div>
  </form>
</div>
```

### Reduced test cases

See above

### What operating system(s) are you seeing the problem on?

n/a

### What browser(s) are you seeing the problem on?

n/a

### What version of Bootstrap are you using?

5.3

**Comment by mdo** [comment]
Not an issue, there's no explicit submit button there. You can mark it up like that, or you can replace any of the `div`s with a form element, but it's not necessary.
